account/views.py

```diff
@@ -1,3 +1,70 @@
-from django.shortcuts import render
+import bcrypt, json, jwt, os, requests, sys, re
 
-# Create your views here.
+from django.views           import View
+from django.shortcuts       import render
+from django.http            import JsonResponse
+from django.core.validators import validate_email
+from django.core.exceptions import ValidationError
+
+from .models      import User
+from my_settings  import SECRET_KEY, ALGORITHM
+
+def validation_user_name(user_name):
+    check = re.search('\W|\s', user_name)
+    if check:
+        return False
+    elif len(user_name) == 0:
+        return False
+    return True
+
+def validation_password(password):
+    if len(password) > 5:
+        return True
+    return False
+
+class SignUpView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        try:
+            vu = validation_user_name(data['user_name'])
+            validate_email(data['email'])
+            vp = validation_password(data['password'])
+            if User.objects.filter(email=data['email']).exists():
+                return JsonResponse({'message':'THIS EMAIL ALREADY EXISTS'}, status=400)
+            elif User.objects.filter(user_name=data['user_name']).exists():
+                return JsonResponse({'message':'THIS USER NAME ALREADY EXISTS'}, status=400)
+            elif (vu and vp) == True:
```

Comment:
`vu and vp`와 같이 값 자체가 boolean일 경우 비교 연산자를 사용하지 않고 바로 사용하는 것이 보다 파이썬스러운 흐름입니다.
```suggestion
            elif (vu and vp):
```

Reply:
제가 착각을 했네요. vu와 vp가 모두 True여야 회원가입을 시키는거라,
(vu and vp and True)로 바꾸겠습니다~